### PR TITLE
renaming folders

### PR DESCRIPTION
**Pull request by Hamza-A**
## Description
This branch aims to make some folder's name plural

- I renamed `command` to `commands`
- I renamed `middleware` to `middlewares`
## Type of change

- [x] **Chore** (non-breaking change which refactors / improves the existing code base).
- [ ] **Bug fix** (non-breaking change which fixes an issue).
- [ ] **New feature** (non-breaking change which adds functionality).
- [ ] **Breaking change** (fix or feature that would cause existing functionality to not work as expected).

## How Has This Been Tested?
There were no need to add or modify tests

## Checklist:

- [x] My code follows the style [**contributing guidelines**][contributing_file] of this project.
- [x] I have performed a self-review of my own code.
- [ ] I have commented my code in hard-to-understand areas.
- [ ] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [ ] I have added tests that prove my fix is effective or that my feature works.

[contributing_file]: https://github.com/fewlinesco/connect-account/blob/master/README.adoc

### CODE REVIEW

File: src/pages/account/index.tsx [hunk 1]
```diff
@@ -7,9 +7,9 @@ import { AccountOverview } from "@src/components/display/fewlines/AccountOvervie
 import { Container } from "@src/components/display/fewlines/Container";
 import { H1 } from "@src/components/display/fewlines/H1/H1";
 import { oauth2Client, config } from "@src/config";
-import { deviceBreakpoints } from "@src/design-system/theme";
-import { withSSRLogger } from "@src/middleware/withSSRLogger";
-import withSession from "@src/middleware/withSession";
+import { deviceBreakpoints } from "@src/design-system/theme/lightTheme";
```
Comment (Fuzznimp) [inline src/pages/account/index.tsx:10]:
```suggestion
import { deviceBreakpoints } from "@src/design-system/theme";
```

File: src/pages/api/identities.ts [hunk 2]
```diff
@@ -1,9 +1,9 @@
 import { HttpStatus } from "@fwl/web";
 
 import { Handler } from "../../@types/ApiPageHandler";
-import { addIdentityToUser } from "../../command/addIdentityToUser";
-import { removeIdentityFromUser } from "../../command/removeIdentityFromUser";
-import { withAPIPageLogger } from "../../middleware/withAPIPageLogger";
+import { addIdentityToUser } from "../../commands/addIdentityToUser";
+import { removeIdentityFromUser } from "../../commands/removeIdentityFromUser";
+import { withAPIPageLogger } from "../../middlewares/withAPIPageLogger";
```
Comment (Fuzznimp) [inline src/pages/api/identities.ts:6]: Can you use path alias here?

**Review by Fuzznimp — CHANGES_REQUESTED** [review]

File: src/pages/api/identities.ts (hunk 2 above)
Comment (Hamza-A) [reply src/pages/api/identities.ts]: OK fixed ccfa47e

Comment (Fuzznimp) [reply src/pages/api/identities.ts]: Hm I'm confused, all I can see are changes I made yesterday in your commit.

Comment (Hamza-A) [reply src/pages/api/identities.ts]: Sorry, it was this commit instead cf9c211

File: src/pages/account/index.tsx (hunk 1 above)
Comment (Hamza-A) [reply src/pages/account/index.tsx]: OK fixed fa08a9e

**Review by Fuzznimp — APPROVED** [review]

**Pull request merged**